app/services/APIService.scala

```diff
@@ -0,0 +1,19 @@
+package services
+
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+import java.time.temporal.ChronoUnit
+
+class APIService {
+}
+
+object APIService{
+  //get DateTime from server
+  def getDateTime: String ={
+    val dateTime: String =  LocalDateTime.now().toString
+    val formatter: DateTimeFormatter = DateTimeFormatter.ISO_LOCAL_DATE_TIME
```

Comment:
notice that this variable is redundant, you can just use `DateTimeFormatter.ISO_LOCAL_DATE_TIME` directly